roles/validate_inventory/defaults/main.yml

```diff
@@ -27,7 +27,8 @@ supported_ocp_versions:
 - 4.6.16
 - 4.7.52
 - 4.8.43
-- 4.9.40
-- 4.10.16
+- 4.9.45
+- 4.10.37
+- 4.11.24
```

Comment:
Missing `4.12.0`

Reply:
See (3).  Have not had a chance to test myself, unless you have?

Reply:
Fair. Lets put it in for now and I'll test it.